src/util/psbt/map/output.rs

```diff
@@ -99,6 +100,13 @@ impl AsRef<TaprootBuilder> for TapTree {
     }
 }
 
+impl Borrow<TaprootBuilder> for TapTree {
```

Comment:
Same comment as the previous commit. Is there any reason to do this? Have we done this for inner types before? I think we should avoid implementing it unless we have a reason. Ref, Deref (IMO) should only be implemented on types that are super-obvious to convert and only if they are really needed. 

Allowing function args that accept these is however another story.

Reply:
There is a reason to do this and I always do that for the new types. `Deref`, otherwise, is not always useful for them.

Some information: https://itfanr.gitbooks.io/rust-doc-en/content/borrow-and-asref.html